### PR TITLE
PIM-5097: Add job configuration warning

### PR DESCRIPTION
**Pull request by SamirBoulil**

### CODE REVIEW

**Comment by juliensnz** [comment]
GTM

File: cookbook/import_export/create-connector.rst [hunk 1]
```diff
@@ -42,6 +42,10 @@ The writer is implemented in the class ``Pim\Component\Connector\Writer\DummyIte
 
 We'll explain in next cookbook chapters how to create your own elements with real logic inside.
 
+.. warning::
+
+   It is also possible to create multiple job configuration files by placing them in a folder ``Resources/config/batch_jobs``.
```
Comment (willy-ahva) [inline cookbook/import_export/create-connector.rst]: `Resources/config/batch_jobs/any_name.yml` or `Resources/config/batch_jobs/xxx.yml` could be more understandable I think.

**Comment by willy-ahva** [comment]
GTM once fix

File: cookbook/import_export/create-connector.rst (hunk 1 above)
Comment (nidup) [inline cookbook/import_export/create-connector.rst]: would use "note" and not "warning" then GTM

Comment (SamirBoulil) [inline cookbook/import_export/create-connector.rst]: @willy-ahva good catch
@nidup understood

File: cookbook/import_export/create-connector.rst [hunk 2]
```diff
@@ -42,6 +42,12 @@ The writer is implemented in the class ``Pim\Component\Connector\Writer\DummyIte
 
 We'll explain in next cookbook chapters how to create your own elements with real logic inside.
 
+.. note::
+
+   It is also possible to create multiple job configuration files by placing them in a folder ``Resources/config/batch_jobs``. The job configurations should be yaml files of any name.
```
Comment (phaseinducer) [inline cookbook/import_export/create-connector.rst]: *must be

File: cookbook/import_export/create-connector.rst [hunk 3]
```diff
@@ -42,6 +42,12 @@ The writer is implemented in the class ``Pim\Component\Connector\Writer\DummyIte
...
+
+   For instance, I can declare a job configuration within this custom file ``Resources/config/batch_jobs/custom_jobs.yml``.
```
Comment (phaseinducer) [inline cookbook/import_export/create-connector.rst]: in a file named... ?

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**